File: TQMall/TQMallWeb/Models/Brand.cs
namespace TQMallWeb.Models
{
  
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;

    public  class Brand
    {
        
        [Key]
        public int ID { get; set; }
        
        public string Name { get; set; }

        public string Image { get; set; }

        public bool? Status { get; set; }
        
      
    }
}

File: DoAnTN/app/src/main/java/com/example/doantn/Models/Brand.java
package com.example.doantn.Models;

public class Brand {
    private int ID;
    private String Name;
    private String Image;
    private boolean Status;

    public int getID() {
        return ID;
    }

    public void setID(int ID) {
        this.ID = ID;
    }

    public String getName() {
        return Name;
    }

    public void setName(String name) {
        Name = name;
    }

    public String getImage() {
        return Image;
    }

    public void setImage(String image) {
        Image = image;
    }

    public boolean isStatus() {
        return Status;
    }

    public void setStatus(boolean status) {
        Status = status;
    }

    public Brand(int ID, String name, String image, boolean status) {
        this.ID = ID;
        Name = name;
        Image = image;
        Status = status;
    }
}
